LedDashboard/Modules/LeagueOfLegends/ChampionModules/AzirModule.cs

```diff
@@ -0,0 +1,136 @@
+                            : base(ledCount, championName, gameState, preferredLightMode)
+        {
+            // Initialization for the champion module occurs here.
+
+            // Set preferred cast mode. It's a player choice (Quick cast, Quick cast with indicator, or Normal cast)
+            PreferredCastMode = preferredCastMode;
+
+            // Set cast modes for abilities.
+            // For Vel'Koz, for example:
+            // Q -> Normal ability, but it can be recast within 1.15s
+            // W -> Normal ability
+            // E -> Normal ability
+            // R -> Instant ability, it is cast the moment the key is pressed, but it can be recast within 2.3s
+            Dictionary<AbilityKey, AbilityCastMode> abilityCastModes = new Dictionary<AbilityKey, AbilityCastMode>()
+            {
+                [AbilityKey.Q] = AbilityCastMode.Instant(),
+                [AbilityKey.W] = AbilityCastMode.Instant(),
+                [AbilityKey.E] = AbilityCastMode.Instant(),
+                [AbilityKey.R] = AbilityCastMode.Instant(),
+            };
+            AbilityCastModes = abilityCastModes;
+
+            // Preload all the animations you'll want to use. MAKE SURE that each animation file
+            // has its Build Action set to "Content" and "Copy to Output Directory" is set to "Always".
+
+            animator.PreloadAnimation(ANIMATION_PATH + "Azir/q_cast.txt");
+            animator.PreloadAnimation(ANIMATION_PATH + "Azir/w_cast.txt");
+            animator.PreloadAnimation(ANIMATION_PATH + "Azir/e_cast.txt");
+            animator.PreloadAnimation(ANIMATION_PATH + "Azir/r_cast.txt");
+
+
+            ChampionInfoLoaded += OnChampionInfoLoaded;
+        }
+
+        /// <summary>
+        /// Called when the champion info has been retrieved.
+        /// </summary>
+        private void OnChampionInfoLoaded(ChampionAttributes champInfo)
+        {
+            animator.NewFrameReady += (_, ls, mode) => DispatchNewFrame(ls, mode);
+            AbilityCast += OnAbilityCast;
+            AbilityRecast += OnAbilityRecast;
+        }
+
+        /// <summary>
+        /// Called when an ability is cast.
+        /// </summary>
+        private void OnAbilityCast(object s, AbilityKey key)
+        {
+            if (key == AbilityKey.Q)
+            {
+                OnCastQ();
+            }
+            if (key == AbilityKey.W)
+            {
+                OnCastW();
+            }
+            if (key == AbilityKey.E)
+            {
+                OnCastE();
+            }
+            if (key == AbilityKey.R)
+            {
+                OnCastR();
+            }
+        }
+
+        private void OnCastQ()
+        {
+            Task.Run(async () =>
+            {
+                animator.RunAnimationOnce(ANIMATION_PATH + "Azir/q_start.txt");
+            });

```

Comment:
Task.Run is not needed because you are not using await on anything (Task.Delay, etc). You can simply call RunAnimationOnce, it's fire and forget.